app/proc/telegram.go

```diff
@@ -20,12 +24,23 @@ const maxTelegramFileSize = 50_000_000
 
 // TelegramClient client
 type TelegramClient struct {
-	Bot     *tb.Bot
-	Timeout time.Duration
+	Bot            *tb.Bot
+	Timeout        time.Duration
+	uploaderConfig *TelegramUploaderConfig
+}
+
+// TelegramUploaderConfig struct used to configure
+// experimental uploader to Telegram for large audio files
+type TelegramUploaderConfig struct {
```

Comment:
TelegramUploaderConfig part can be inlined with a namespace and "uploader_" prefix can be removed from the param. See a similar thing here https://github.com/radio-t/super-bot/blob/master/app/main.go#L23

Reply:
Done